homeassistant/components/plant/__init__.py

```diff
@@ -133,38 +126,38 @@ class Plant(Entity):
     _attr_should_poll = False
 
     READINGS = {
-        READING_BATTERY: {
+        BATTERY_SENSOR: {
             ATTR_UNIT_OF_MEASUREMENT: PERCENTAGE,
             "min": CONF_MIN_BATTERY_LEVEL,
         },
-        READING_TEMPERATURE: {
+        TEMPERATURE_SENSOR: {
             ATTR_UNIT_OF_MEASUREMENT: UnitOfTemperature.CELSIUS,
             "min": CONF_MIN_TEMPERATURE,
             "max": CONF_MAX_TEMPERATURE,
         },
-        READING_MOISTURE: {
+        MOISTURE_SENSOR: {
             ATTR_UNIT_OF_MEASUREMENT: PERCENTAGE,
             "min": CONF_MIN_MOISTURE,
             "max": CONF_MAX_MOISTURE,
         },
-        READING_CONDUCTIVITY: {
+        CONDUCTIVITY_SENSOR: {
             ATTR_UNIT_OF_MEASUREMENT: CONDUCTIVITY,
             "min": CONF_MIN_CONDUCTIVITY,
             "max": CONF_MAX_CONDUCTIVITY,
         },
-        READING_BRIGHTNESS: {
+        BRIGHTNESS_SENSOR: {
             ATTR_UNIT_OF_MEASUREMENT: LIGHT_LUX,
             "min": CONF_MIN_BRIGHTNESS,
             "max": CONF_MAX_BRIGHTNESS,
         },
     }
 
-    def __init__(self, name, config):
+    def __init__(self, name, config) -> None:
```

Comment:
Unrelated to the move of the constants (and all arguments would need to be typed).